firebase/package.json

```diff
@@ -40,12 +40,15 @@
     "@types/sinon": "^10.0.x",
     "@typescript-eslint/eslint-plugin": "^4.32.x",
     "@typescript-eslint/parser": "^4.32.x",
+    "@types/hapi__joi": "^17.1.7",
     "ajv": "^8.8.2",
+    "ajv-formats": "^2.1.1",
     "eslint": "^7.32.x",
     "eslint-config-prettier": "^8.3.x",
     "eslint-plugin-prettier": "^4.0.x",
     "firebase-admin": "^9.11.x",
     "firebase-tools": "^9.18.x",
+    "joi": "17.4.2",
```

Comment:
This is a common-case for the suggestions I've made in https://github.com/olmps/memo/pull/255. This `joi` dependency must have the same version as the inner `Joi` dependency (src/core/package.json), otherwise the tests don't compile.